src/components/app/ServiceWorkerManager.js

```diff
@@ -212,7 +212,12 @@ class ServiceWorkerManagerImpl extends PureComponent<Props, State> {
   }
 
   componentDidMount() {
-    if (process.env.NODE_ENV === 'production') {
+    if (
+      process.env.NODE_ENV === 'production' &&
+      // Do not install the service worker for l10n branch so localizers can see
+      // the changes easily with a single refresh.
```

Comment:
nit: maybe add another line just explaining that Webpack changes this on the fly?